src/main/java/org/owasp/validator/html/Policy.java

```diff
@@ -276,6 +276,19 @@ private static Element getTopLevelElement(InputStream is) throws PolicyException
     protected static Element getTopLevelElement(InputSource source) throws PolicyException {
         try {
             DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+            /**
+             * Disable external entities, etc.
+             */
+            String FEATURE = null;
```

Comment:
The re-use of this FEATURE variable for me actually makes this harder to read. I'd rather see literals or constants.